addons/binding/org.openhab.binding.kostalinverter/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:kostalplenticorepikoiq_config">
```

Comment:
Please use the config description of the thing already present. Probably copy it to here.
You should keep the `context` `network-address` from your config.
I noticed the original thing has default values for username/password. If these don't apply to your things. The easiest way is to do use a different config, just try keep it the same  as much as possible.

Reply:
I used the description for the refresh timer.
The other config settings do not really match
(no username, no URL (just hostname), password does not have any description to re-use)

will be in the next commit